lighthouse-cli/test/cli/index-test.js

```diff
@@ -6,21 +6,24 @@
 'use strict';
 
 /* eslint-env jest */
-const assert = require('assert').strict;
-const childProcess = require('child_process');
-const path = require('path');
-const indexPath = path.resolve(__dirname, '../../index.js');
-const spawnSync = childProcess.spawnSync;
+
+import {strict as assert} from 'assert';
+import {spawnSync} from 'child_process';
+
+import {LH_ROOT} from '../../../root.js';
+
+const indexPath = `${LH_ROOT}/lighthouse-cli/index.js`;
 
 describe('CLI Tests', function() {
   it('fails if a url is not provided', () => {
-    const ret = spawnSync('node', [indexPath], {encoding: 'utf8'});
+    const ret = spawnSync('node', ['--experimental-vm-modules', indexPath], {encoding: 'utf8'});
```

Comment:
Shouldn't `--experimental-vm-modules` not be needed since it's the jest VM that's spawning the child process, but the child process isn't jest?

Reply:
ya